src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -2268,6 +2268,13 @@ def resolve_api_version(rcf, resource_provider_namespace, parent_resource_path,
     @staticmethod
     def _resolve_api_version_by_id(rcf, resource_id):
         parts = parse_resource_id(resource_id)
+
+        if len(parts) == 2 and parts['subscription'] is not None and parts['resource_group'] is not None:
+            return AZURE_API_PROFILES['latest'][ResourceType.MGMT_RESOURCE_RESOURCES]
```

Comment:
I'd prefer to add multi-API support in `azure.cli.command_modules.resource.custom._ResourceUtils._resolve_api_version_by_id` for resource group operations and use corresponding API versions for `ResourceType.MGMT_RESOURCE_RESOURCES`.

Reply:
`az resource tag` has the `--api-version` parameter which provider the api version of the resource (omit for latest version).